woocommerce/woocommerce-gutenberg-products-block
# Add `Featured Categories Grid` pattern

## What

Adds the new `Featured Categories Grid` pattern.

## Why

Fixes https://github.com/woocommerce/woocommerce/issues/42241

## Testing Instructions

1. Create a new page or post.
2. Insert the `Featured Categories Grid` pattern.
3. Check it has 3 items per row and all the Product Categories are present.
4. Save, and check it looks good also con the front end.


## Screenshots or screencast

<img width="992" alt="Screenshot 2023-09-01 at 12 31 36" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/3d0d6a2f-2a36-48a7-89e0-d81d92cfa1c8">

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Adds the new `Featured Categories Grid` pattern.
